src/util/objects.ts

```diff
@@ -99,7 +99,7 @@ export interface CloneOptions {
 }
 
 export function deepClone (obj: any, options: CloneOptions = {}) {
-	if (!obj) {
+	if (!obj || typeof obj !== 'object') {
```

Comment:
We should consider primitive values as well. If not, if we get, for example, `true` as a value of `obj`, we'll try to use it as a `_clones` key, and the code will throw.